tee-worker/local-setup/github-action-config-one-worker.json

```diff
@@ -16,7 +16,8 @@
         "mock",
         "--enable-mock-server",
         "--parentchain-start-block",
-        "0"
+        "0", 
+	"--enable-metrics"
```

Comment:
please correct formatting here

Reply:
^ this :)

And what about other config jsons?